src/main/features/claim/views/interest-date.njk

```diff
@@ -38,7 +38,15 @@
               {{ textInput('Explain why you’re claiming from this date', 'reason', form) }}
 
             </div>
-            {{ radioOption('Claim interest from the date I submit the claim', 'type', form, 'submission', ' ', form.valueFor('type') == 'submission') }}
+            {{ radioOption(
+              label = 'Claim interest from the date I submit the claim',
+              name = 'type',
+              form = form,
+              value = 'submission',
+              hint = ' ',
+              checked = form.valueFor('type') == 'submission',
```

Comment:
why don't we use const for `submission`?

Reply:
Good catch, I just reformatted this bit and didn't have a closer look. Will do that 👍